apps/backend/api/src/app/characters/character.service.ts

```diff
@@ -10,19 +10,41 @@ export class CharactersService {
   constructor(
     @InjectRepository(Character)
     private charactersRepository: Repository<Character>,
+    @InjectRepository(Room)
+    private roomsRepository: Repository<Room>,
   ) {}
 
   async create(createCharacterDto: CreateCharacterDto) {
-    return this.charactersRepository.save(createCharacterDto);
+    const room = await this.roomsRepository
+      .createQueryBuilder('room')
+      .where('room.id = :id', { id: createCharacterDto.roomId })
+      .getOne();
+
+    const errors: { errorGame?: string; errorRoom?: string } = {};
+
+    if (room) {
+      this.charactersRepository.save({
```

Comment:
Il manque `await` ici
Vérifier tous les autres services au même endroit

Tu peux les trouver en cherchant
`[\n\r\t]+\s{0,}(this\.)+(.)*(\.save)` dans VSCode avec le filtre regex

Reply:
Fait